python-regression/tests/features/steps/response_handling_steps.py

```diff
@@ -24,6 +24,8 @@ def compare_thread_return(step, api_call):
     # Exclude duration from response list
     if 'duration' in response_list:
         del response_list['duration']
+    if 'info' in response_list:
```

Comment:
Whats this for dyrell?

Reply:
This is for checkConsistency negative responses. The length of arguments wasn't matching because info was not included in the expected responses list